packages/esm-commons-lib/src/api/api.ts

```diff
@@ -239,3 +239,23 @@ export function fetchFormsClobData(valueReferences: string[]) {
   const fetch = (ref: string) => openmrsFetch(`/ws/rest/v1/clobdata/${ref}`);
   return Promise.all(valueReferences?.map((ref) => fetch(ref)));
 }
+
+export async function fetchMambaReportData(reportId: string) {
+  try {
+    const response = await openmrsFetch(`ws/rest/v1/mamba/report?report_id=${reportId}`);
+    const data = await response.json();
+
```

Comment:
@pirupius, I have moved  this to the commons-lib and rename it to fetchMambaReportData